src/leap/keymanager/__init__.py

```diff
@@ -922,6 +922,5 @@ def _get_domain(url):
     """
     return urlparse(url).hostname
 
-from ._version import get_versions
 __version__ = get_versions()['version']
 del get_versions
```

Comment:
This code doesn't work, as in line 87 `get_versions` get deleted. I don't understand why the set of `__version__` is duplicated in this code. I'll be happy to remove all this block at the end of the file and just keep the version setting on the top.
```
Traceback (most recent call last):
  File "/home/meskio/dev/leap/env/local/lib/python2.7/site-packages/twisted/trial/runner.py", line 605, in loadByNames
    things.append(self.findByName(name))
  File "/home/meskio/dev/leap/env/local/lib/python2.7/site-packages/twisted/trial/runner.py", line 409, in findByName
    return reflect.namedAny(name)
  File "/home/meskio/dev/leap/env/local/lib/python2.7/site-packages/twisted/python/reflect.py", line 303, in namedAny
    topLevelPackage = _importAndCheckStack(trialname)
  File "/home/meskio/dev/leap/env/local/lib/python2.7/site-packages/twisted/python/reflect.py", line 242, in _importAndCheckStack
    return __import__(importName)
  File "/home/meskio/dev/leap/keymanager/src/leap/keymanager/__init__.py", line 925, in <module>
    __version__ = get_versions()['version']
exceptions.NameError: name 'get_versions' is not defined
```